doc/tutorials/imgproc/pyramids/pyramids.markdown

```diff
@@ -105,15 +105,15 @@ Let's check the general structure of the program:
     while( true )
     {
       int c;
-      c = waitKey(10);
+      c = waitKey(10) & 0xFF;
 
-      if( (char)c == 27 )
+      if( c == 27 )
```

Comment:
Original PR fixes warning (compare int and char)), which is added back by changes like these.